tsconfig.json

```diff
@@ -14,6 +15,8 @@
     "lib": [
       "es2017",
       "dom"
-    ]
+    ],
+    "module": "es2015",
```

Comment:
What does `module` option do?

Reply:
this change is made by ng update command.
I also checked by removing this change but the same error is coming.